ui/server/src/main/scala/pl/touk/nussknacker/ui/api/DefinitionResources.scala

```diff
@@ -25,7 +25,7 @@ class DefinitionResources(modelData: Map[ProcessingType, ModelData],
     //TODO maybe always return data for all subprocesses versions instead of fetching just one-by-one?
     path("processDefinitionData" / Segment) { (processingType) =>
       parameter('isSubprocess.as[Boolean]) { (isSubprocess) =>
-        post {
+        post { // POST - because there is complicated logic of retrieving data
```

Comment:
Logic of retrieving is not complicated. Only query parameter (subprocessVersions) is complex and it is the reason why POST was used

Reply:
Changed.